core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/dot-ema-shell.component.scss

```diff
@@ -37,4 +37,10 @@
             padding-top: 0;
         }
     }
+
+    dot-experiments-shell dot-experiments-list {
+        .dot-experiment {
+            padding: $spacing-3 $spacing-5 $spacing-5 $spacing-5 !important;
```

Comment:
is possible to remove the `!important` with specificity ?

Reply:
I tried to and I couldn't